README.md

```diff
@@ -13,8 +13,17 @@ The logging client provides four things:
 ### Local Development
 * Install Python dependencies: `pipenv install --dev`
 * Run the test suite: `pytest`
-* Run lint checks: `pytest --flakes`
-* Auto-format: `black .`
+* Run lint checks: `ruff .`
+* Auto-format: `ruff format`
+
+#### Testing
+
+We use `moto` to test the various AWS moving parts. `moto`
+builds (almost) everything from the CDK CloudFormation template. 
+
+If you make change to the Stack, or the first time you install the project, you 
+need to run `make cfn_template_for_tests`. This file isn't checked in to Git
+as it contains actual values from the deployment. 
```

Comment:
I did consider this, but then we'd have to remember that this project doesn't use `pytest`, or rather that's not how it's invoked. I think that's more confusing for devs. If the file isn't there, the tests aren't going to run.

I could add a call _inside_ `pytest` to call `make` before it runs test. This would mostly be a no-op if the cloudformation YAML was up to date locally?

Reply:
Yeah, that makes sense to leave it. I think altering pytest to call make could also get confusing!